app/models/grda_warehouse/tasks/push_clients_to_cas.rb

```diff
@@ -115,6 +117,7 @@ def client_source
       GrdaWarehouse::Hud::Client.cas_active
     end
 
+    # TODO: update so all things coming from HMIS assessments or ETO get a wrapper method that checks assessments first and then client attribute
```

Comment:
Is this satisfied by the Calculator classes?

Reply:
Yes, but maybe not completely, so I left it as a reminder.